### DISCUSSION TITLE
Pipewire editor

### DISCUSSION THREAD

**Discussion by MrDorianJames** [pop-os/cosmic-epoch#1053] [Ideas]
I know this is far down the list but it would be nice to have a rust-based pipewire editor for cosmic that is accessible through both the settings and the audio applet. Something like configure button. 

Someone started building a rust based editor but it seems abandoned. [https://github.com/Ax9D/pw-viz](https://github.com/Ax9D/pw-viz)

**Comment by matdave** [comment]
It doesn't look abandoned, just slow updates.
